### PR TITLE
Fix spelling, grammar, javadocs, and similar issues

### PR DESCRIPTION
**Pull request by fooooooooooooooo**
Sorry for the large amount of changes.
I did not change any NBT or config keys, or any item names.
I changed only variable/parameter/field/class names and added types to javadocs
No worries if any changes are required

### CODE REVIEW

**Review by modmuss50 — APPROVED** [review]
Wow! thanks for fixing my bad spelling 👍

File: RebornCore/src/main/java/reborncore/common/fluid/RebornFluidRenderManager.java [hunk 1]
```diff
@@ -56,7 +56,8 @@ public static void setupClient() {
 	}
 
 	private static void setupFluidRenderer(RebornFluid fluid) {
-		//Done lazy as we want to ensure we get the sprite at the correct time, but also dont want to be making these calls every time its required.
+		// Done lazy as we want to ensure we get the sprite at the correct time,
+	    // but also don't want to be making these calls every time its required.
```
Comment (Ayutac) [inline RebornCore/src/main/java/reborncore/common/fluid/RebornFluidRenderManager.java:60]: wrongly intended

File: RebornCore/src/main/java/reborncore/common/multiblock/MultiblockBlockEntityBase.java [hunk 2]
```diff
@@ -334,7 +339,7 @@ public void onOrphaned(MultiblockControllerBase controller, int oldSize, int new
 		getWorld().markDirty(getPos());
 	}
 
-	// // Helper functions for notifying neighboring blocks
+	// /// Helper functions for notifying neighboring blocks
```
Comment (Ayutac) [inline RebornCore/src/main/java/reborncore/common/multiblock/MultiblockBlockEntityBase.java:342]: what is this, lol?

File: RebornCore/src/main/java/reborncore/common/multiblock/rectangular/RectangularMultiblockBlockEntityBase.java [hunk 3]
```diff
@@ -121,7 +121,7 @@ public void recalculateOutwardsDirection(BlockPos minCoord, BlockPos maxCoord) {
 		}
 	}
 
-	// /// Validation Helpers (IMultiblockPart)
+	// Validation Helpers (IMultiblockPart)
```
Comment (Ayutac) [inline RebornCore/src/main/java/reborncore/common/multiblock/rectangular/RectangularMultiblockBlockEntityBase.java:124]: this seems like a wrong correction

File: RebornCore/src/main/java/reborncore/common/network/ServerBoundPackets.java [hunk 4]
```diff
@@ -54,7 +54,8 @@ public static void init() {
 				IdentifiedPacket packetFluidConfigSync = ClientBoundPackets.createPacketFluidConfigSync(pos, legacyMachineBase.fluidConfiguration);
 				NetworkManager.sendToTracking(packetFluidConfigSync, legacyMachineBase);
 
-				//We update the block to allow pipes that are connecting to detctect the update and change their connection status if needed
+				// We update the block to allow pipes that are connecting to detect the update and change their
+			    // connection status if needed
```
Comment (Ayutac) [inline RebornCore/src/main/java/reborncore/common/network/ServerBoundPackets.java:58]: wrongly intended

File: RebornCore/src/main/java/reborncore/common/powerSystem/PowerAcceptorBlockEntity.java [hunk 5]
```diff
@@ -232,62 +233,68 @@ protected boolean canAcceptEnergy(@Nullable Direction side){
...
+	 * Wrapper method used to sync energy values with client via
+	 * {@link BlockEntityScreenHandlerBuilder}
 	 *
-	 * @return double Energy stored in block entity
+	 * @return {@code long} Energy stored in block entity
 	 */
-
 	public long getEnergy() {
-		return getStored();
+	  return getStored();
```
Comment (Ayutac) [inline RebornCore/src/main/java/reborncore/common/powerSystem/PowerAcceptorBlockEntity.java:290]: wrong intended

File: RebornCore/src/main/java/reborncore/common/recipes/RecipeCrafter.java [hunk 6]
```diff
@@ -354,16 +354,17 @@ public boolean isInvDirty() {
...
 	}
 
 	public boolean isStackValidInput(ItemStack stack) {
 		if (stack.isEmpty()) {
 			return false;
 		}
 
-		//Test with a stack with the max stack size as some independents will check the stacksize. Bit of a hack but should work.
+		// Test with a stack with the max stack size as some independents will check the stack size.
+	    // A bit of a hack but should work.
```
Comment (Ayutac) [inline RebornCore/src/main/java/reborncore/common/recipes/RecipeCrafter.java:367]: wrongly intended

**Comment by Ayutac** [comment]
I finally had time to read this commit in detail. I acknowledge that no code functionality was changed as far as I could see.

File: RebornCore/src/main/java/reborncore/common/multiblock/rectangular/RectangularMultiblockBlockEntityBase.java (hunk 3 above)
Comment (fooooooooooooooo) [reply RebornCore/src/main/java/reborncore/common/multiblock/rectangular/RectangularMultiblockBlockEntityBase.java:124]: accidentally removed these, then reverted, guess i missed this one

**Pull request merged**